src/wallet.js

```diff
@@ -1746,6 +1771,21 @@ const wallet = {
     return this.decryptData(accessData.words, password);
   },
 
+  /**
+   * Get the privKey of the auth derivation
+   *
+   * @param {string} pin Pin to decrypt the words
+   *
+   * @return {string} The auth private key
```

Comment:
```suggestion
   * @return {HDPrivateKey} The auth xpriv
```

Reply:
Thanks! Fixed in 7be2bddfaa7280addf82144f2b3ef67767fac655